core/templates/dev/head/services/context.service.ts

```diff
@@ -109,6 +110,20 @@ export class ContextService {
     }
   }
 
+  isContextLessonRelated(): boolean {
+    return (
+      this.getPageContext() === ServicesConstants.PAGE_CONTEXT.TOPIC_EDITOR ||
+      this.getPageContext() === ServicesConstants.PAGE_CONTEXT.SKILL_EDITOR ||
+      this.getPageContext() ===
+        ServicesConstants.PAGE_CONTEXT.EXPLORATION_EDITOR &&
+        this.contextIsLessonRelated
```

Comment:
This is confusing because one would expect this.contextIslessonRelated to be the same as isContextLessonRelated() in all cases, just by the naming.

It seems like this.contextIsLessonRelated and isContextLessonRelated() express different concepts. Could you choose more distinct names for them?

Reply:
Done